src/main/java/com/rabbitmq/client/ChannelContinuationTimeoutException.java

```diff
@@ -0,0 +1,35 @@
+package com.rabbitmq.client;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Exception thrown when a channel times out on a continuation during a RPC call.
+ * @since 4.1.0
+ */
+public class ChannelContinuationTimeoutException extends IOException {
+
+    /**
+     * The channel that performed the call.
+     */
+    private final Object channel;
```

Comment:
Let's add a channel number field (we can keep the field we have with a comment).